contracts/Core/RewardManager.sol

```diff
@@ -126,7 +126,7 @@ contract RewardManager is Initializable, ACL, Constants {
         if (mediansLastEpoch.length == 0) return;
         uint64 penalty = 0;
         for (uint8 i = 0; i < mediansLastEpoch.length; i++) {
-            uint32 voteValueLastEpoch = voteManager.getVoteValue(i, stakerId);
+            uint32 voteValueLastEpoch = voteManager.getVoteValue(i + 1, stakerId);
```

Comment:
start the value of i from 1 instead.

Reply:
sure we can do so, but if we do it here 
on line 131 we will have to do -1
In the end it remains the same 

so should we do it?